releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd

```diff
@@ -27,6 +27,7 @@ location "http://download.eclipse.org/tools/orbit/downloads/drops/S2016091616200
 	org.antlr.runtime [4.3.0,4.3.1) // CQ 11787
 	com.google.inject [3.0.0,3.0.1) // CQ 11788
 	org.apache.log4j [1.2.15,1.2.16) // CQ 11789
+	javax.measure.unit-api [1.0.0] // CQ 13959
```

Comment:
The CQ is done and the package will soon be in Orbit.

The location above needs to be updated to the appropriate repo once there is one built. Initially we can use an I build, but should change to S and then R build as they become available.

Reply:
I created a new pull-request : https://github.com/eclipse/january/pull/255 with the correct signed-off-by section in the commit message so the IP-validation passes.